frontend/src/components/common/Project.tsx

```diff
@@ -59,15 +59,17 @@ const Project = ({ description, issues, language, link, name, stars, type }: Pro
             </a>
           </h1>
           <p style={style.p}>{displayDescription}</p>
-          <div className="d-flex flex-column flex-sm-row justify-content-start flex-wrap">
-            <div className="project-language my-auto">{language || defaultLanguage}</div>
-            <div className="project-language my-auto"> Issues {issues}</div>
-            <div className="d-flex justify-content-start align-items-center">
-              <img alt="star icon" src={themeContext.theme === DARK ? starLightIcon : starIcon} />
-              &ensp;
-              <span className="font-weight-bold">{stars}</span>
+          {language?.length > 0 && issues > 0 && stars > 0 ? (
```

Comment:
I think we would  want to display projects with `0 issue` as that might happen very often, same thing with 0 stars. Regarding language, I am not sure but it is might possible to our backend to return us an open source project with no language specified. I doubt we should keep this conditions